beacon-chain/node/node_test.go

```diff
@@ -47,3 +48,31 @@ func TestNodeClose_OK(t *testing.T) {
 		t.Log(err)
 	}
 }
+func TestBootStrapNodeFile(t *testing.T) {
+	if _, err := os.Stat("sampleNodes.enr"); os.IsExist(err) {
```

Comment:
This would end up polluting our root directory whenever we ran a test. Please use the temp test directory
```go
	file, err := ioutil.TempFile(testutil.TempDir(), "bootstrapFile")
	if err != nil {
		log.Fatal(err)
	}
	defer func() {
		if err := os.Remove(file.Name()); err != nil {
			t.Log(err)
		}
	}()
```

Reply:
Thanks!  I will look into this and adjust appropriately.

Reply:
[Latest commit](https://github.com/prysmaticlabs/prysm/pull/6351/commits/0492cad38991b32f2907c106adf1cc40e88d4e08) should now address this